src/riak_kv_compile_tab.erl

```diff
@@ -167,13 +167,14 @@ get_ddl(BucketType) when is_binary(BucketType) ->
     end.
 
 %%
--spec get_all_compiled_ddls() -> term() | notfound.
-get_all_compiled_ddls() ->
-    case dets:match(?TABLE, {'_','_','$1','_','compiled'}) of
+-spec get_all_table_names() -> term() | notfound.
+get_all_table_names() ->
+    case dets:match(?TABLE, {'$1','_','_','_','compiled'}) of
```

Comment:
Does this have to be a case expression, it is safe for a list comprehension to iterate an empty list.